registries/composition_discriminator.hocon

```diff
@@ -43,7 +46,7 @@ You will receive a problem statement along with a number of suggested solutions
 
             "instructions": """
 You are a solution discriminator.
-You receive a problem, 'P' and a numbered list of its solutions,
+You receive a problem, 'P' and an ordered list of its solutions,
```

Comment:
Tell about the ordered list. It's no longer numbered, but it the lack of numbering becomes a problem we can change the prototype above to have an index as well.  Seems redundant at this time and things are passing.